docs/DeveloperGuide.adoc

```diff
@@ -415,66 +415,78 @@ See <<UsingTravis#deploying-github-pages, UsingTravis.adoc>> to learn how to dep
 
 We use https://www.google.com/chrome/browser/desktop/[Google Chrome] for converting documentation to PDF format, as Chrome's PDF engine preserves hyperlinks used in webpages.
 
-Here are the steps to convert the project documentation files to PDF format.
+Here are the steps to convert the project documentation files to PDF format:
 
 .  Follow the instructions in <<UsingGradle#rendering-asciidoc-files, UsingGradle.adoc>> to convert the AsciiDoc files in the `docs/` directory to HTML format.
-.  Go to your generated HTML files in the `build/docs` folder, right click on them and select `Open with` -> `Google Chrome`.
-.  Within Chrome, click on the `Print` option in Chrome's menu.
-.  Set the destination to `Save as PDF`, then click `Save` to save a copy of the file in PDF format. For best results, use the settings indicated in the screenshot below.
+.  Go to `build/docs` folder, your should see the generated HTML files.
+.  Right click on the file and select `Open with` -> `Google Chrome`. You should see a Google Chrome window.
+.  Click on the `Print` option in Chrome's menu. You should see a `Print` window as shown in Figure 11 below.
+.  Set the destination to `Save as PDF`.
+.  Click `Save` to save a copy of the file in PDF format. For the best results, use the settings indicated in Figure 11.
 
-.Saving documentation as PDF files in Chrome
+.Saving Documentation as PDF Files in Chrome
 image::chrome_save_as_pdf.png[width="300"]
 
 [[Testing]]
 == Testing
 
+This section describes how to run tests, different types of tests and troubleshooting of tests.
```

Comment:
`This section explains how to run tests, the different types of tests, and test troubleshooting` sounds better